src/main/java/window/FileSelector.java

```diff
@@ -1,9 +1,19 @@
 package window;
 
+import filesystem.FileSystem;
 import javafx.stage.FileChooser;
```

Comment:
If you group the import, group all of the imports with each other. Now the two javafx imports are separated.